functions/test/userToken.ts

```diff
@@ -0,0 +1 @@
+export const token = "put expired token here";
```

Comment:
why is this separate file needed just for an exported const? if it is only used in one file why not just inline it there?